extensions/workouts/src/utils.ts

```diff
@@ -64,7 +62,7 @@ export const generateMapboxImage = (polyline: string) => {
   const padding = encodeURIComponent("40,40,80");
   const poly = encodeURIComponent(polyline);
   const mapboxStyle = environment.appearance === "light" ? "outdoors-v12" : "dark-v11";
-  return `https://api.mapbox.com/styles/v1/mapbox/${mapboxStyle}/static/path-5+f44-0.5(${poly})/auto/${width}x${height}?padding=${padding}&access_token=${preferences.mapbox_access_token}`;
+  return `https://extensions-api-proxy.raycast.com/mapbox/styles/v1/mapbox/${mapboxStyle}/static/path-5+f44-0.5(${poly})/auto/${width}x${height}?padding=${padding}`;
```

Comment:
@samuelkraft there's some vpn/firewalls that block connections to URLs that contains "proxy" in them. We had to change the oauth proxy endpoint because of that

Reply:
What's the new url?

Reply:
Now it's oauth.raycast.com - it used to be oauth-proxy.raycast.com and it was blocked by certain firewalls